### PR TITLE
Content changes for contact

### PR DESCRIPTION
**Pull request by smcdonaldea**
https://eaflood.atlassian.net/browse/RL-1256
https://eaflood.atlassian.net/browse/RL-1283
https://eaflood.atlassian.net/browse/RL-1254

New functionality added for contact screen 1284. Content and styling only for 1256 and 1283.

### CODE REVIEW

File: server/models/contact.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
   this.subscribe = contact.subscribe
-  this.errors = error && helper.convertErrors(error, content.errors)
+  this.message = message
   this.dpaReturnUrl = '/buy/contact'
+  this.options = helper
+    .createViewOptions(content, enumeration)
+    .createRadioOptions('contact', contact.contact)
+  this.options.email = contact.email
+  this.options.mobile = contact.mobile
+  var errors = error && helper.convertErrors(error, content.errors)
```
Comment (davidjamesstone) [inline server/models/contact.js]: Use `const`

File: server/routes/buy/contact.js [hunk 2]
```diff
@@ -5,42 +5,70 @@ const sessionHelper = require('../../session')
...
   get: function (request, reply) {
     // Prepare the model
     const session = request.sessionCache
     const contact = session.buy.holder.contact
-
-    const model = new ViewModel(contact)
-
+    // select contact message based on licence type
+    const message = chooseContactMessage(session)
+    var model = new ViewModel(contact, message)
```
Comment (davidjamesstone) [inline server/routes/buy/contact.js]: Use `const`

File: server/routes/buy/contact.js [hunk 3]
```diff
@@ -5,42 +5,70 @@ const sessionHelper = require('../../session')
...
-
-    // Respond with the view
-    return reply.view('contact', model)
+    // get the session as it is not available by default in the failAction handler
+    sessionCache.get(request, function (err, session) {
+      if (err) {
+        return reply(Boom.badRequest('Failed to get the session', err))
+      } else {
+        const message = chooseContactMessage(session)
+        var model = new ViewModel(payload, message, errors)
```
Comment (davidjamesstone) [inline server/routes/buy/contact.js]: Use `const`

File: server/schema/enum/contact.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const Enum = require('.')
+
+class StartKindEnum extends Enum {
```
Comment (davidjamesstone) [inline server/schema/enum/contact.js]: StartKindEnum needs renaming

File: server/routes/buy/contact.js [hunk 5]
```diff
@@ -5,42 +5,70 @@ const sessionHelper = require('../../session')
...
   },
   fail: function (request, reply, source, error) {
-    // Get the errors and prepare the model
     const errors = error.data.details
     const payload = request.payload || {}
-    const model = new ViewModel(payload, errors)
-
-    // Respond with the view
-    return reply.view('contact', model)
+    // get the session as it is not available by default in the failAction handler
```
Comment (davidjamesstone) [inline server/routes/buy/contact.js]: Minor point but could we make comments sentance case (uppercase letter first)

**Comment by smcdonaldea** [comment]
Hi Dave,
Not a problem. I'll make the housekeeping changes and I can run through the contact form with you tomorrow. 
Thanks,
Simon

**Comment by smcdonaldea** [comment]
1256 and 1283 dropped and put onto separate branches.

**Pull request merged**